src/types/interfaces.ts

```diff
@@ -261,3 +261,14 @@ export type MediaStat = InsightAction<{
 
 // This will be a union of all the stats we want to proxy
 export type NRProxyStat = FirstAlertingConversationStat | MediaStat;
+
+export type SCConnectionData = {
```

Comment:
I think this is pretty minor, but I noticed that the order of these fields in `increaseBackoff` seemed to make more sense to me. So I propose changing the order here to match, as follows:

```
currentDelayMs: number;
delayMsAfterNextReduction?: number;
// At this time, we will reduce the current delay.
// This is a long date
nextDelayReductionTime?: number;
// At this time, we will disregard any saved delays. It should be updated every failure.
// This is a long date
timeOfTotalReset?: number;
```

I think it is worth making them consistent, even if we decide a different order is better.